src/app/utility/new-data-type-inline/new-data-type-inline.component.ts

```diff
@@ -70,6 +72,7 @@ export class NewDataTypeInlineComponent implements OnInit, AfterViewInit {
 
   ngOnInit() {
     this.validate();
+    console.log(this.parentDataModel);
```

Comment:
Once you're happy with how it works and tested, this `console.log()` line can be removed.

Reply:
Yep, good spot